app/build.gradle

```diff
@@ -93,6 +94,50 @@ android {
     }
 }
 
+tasks.register('codeCoverage', JacocoReport) {
+    dependsOn 'testDebugUnitTest'
```

Comment:
dependsOn means that it will execute testDebugUnitTest when triggering codeCoverage and then all that is defined here